package.json

```diff
@@ -38,6 +38,7 @@
     "@babel/plugin-transform-runtime": "^7.12.1",
     "@babel/preset-env": "^7.12.1",
     "@babel/register": "^7.12.10",
+    "@hapi/joi": "^17.1.1",
```

Comment:
Why having two package to do one thing? I think since we were validating using `joi` it wasn't necessary to install `@hapi/joi` for validation too.

Reply:
Noted, thanks